edi-update-process.Rmd

```diff
@@ -0,0 +1,113 @@
+---
+title: "Feather River Redd Data EDI Upload Workflow"
+author: "Badhia Yunes Katz (FlowWest)"
+output: html_document
+---
+
+This guide provides step-by-step instructions to update the Feather River Redd Data EDI package with new data.
+
+## Navigating to the Repository
+
+All data processing and EDI uploads are managed in the same repository.  
+
+1. Navigate to the [project repository](https://github.com/FlowWest/feather-redd/tree/main) and clone it to your local machine.
+
+## Adding Data
+
+1. Before adding the data to the repository, make sure the data contains the following fields:
+
+    * date  
+    * type  
+    * number_redds  
+    * number_salmon  
+    * latitude_m_n  
+    * longitude_m_e  
+    * depth_m  
+    * pot_depth_m  
+    * velocity_m_s  
+    * percent_fines  
+    * percent_small  
+    * percent_med  
+    * percent_large  
+    * percent_boulder  
+    * redd_width_m  
+    * redd_length_m
+
+
+2. Add the Excel file with new data into the `dwr_chinook_redd_survey_data` [folder](https://github.com/FlowWest/feather-redd/tree/main/data-raw/dwr_chinook_redd_survey_data) of your local computer copy, following this naming convention: `yyyy_Chinook_Redd_Survey_Data.xlsx`, replacing "yyyy" with the actual year of the data.
+
```

Comment:
Clarify this more. If the format is not the same as x file (you could reference last years) then the script wont work